src/arch/svm/SpokeUtils.ts

```diff
@@ -118,6 +160,16 @@ async function _callGetTimestampForSlotWithRetry(
 
     const { __code: code } = err.context;
     const slot = slotNumber.toString();
+
+    logger.debug({
+      at: "getTimestampForSlot",
+      message: "Caught error from getBlockTime()",
+      errorCode: code,
+      slot,
+      retryAttempt,
+      maxRetries,
+    });
```

Comment:
I think this will be quite noisy for errors where we already have good handling, and it'll make it harder to find the actual unhandled cases. Should we skip logging here and instead log under the `default` case?

Reply:
that's a good call sure